client/client.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	"context"
+
+	sdkparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	"github.com/umee-network/umee/v4/sdkclient"
+)
+
+type Client struct {
+	sdkclient.Client
```

Comment:
I like this idea, I think this is one of the better ways in Golang to mimic inheritance.

Reply:
yes!